vulnerabilities/tests/test_nvd.py

```diff
@@ -22,148 +22,29 @@
 
 import json
 import os
-from unittest import TestCase
 
-from dateutil import parser as dateparser
-
-from vulnerabilities.importer import Advisory
-from vulnerabilities.importer import Reference
-from vulnerabilities.importer import VulnerabilitySeverity
-from vulnerabilities.importers import NVDImporter
-from vulnerabilities.severity_systems import scoring_systems
+from vulnerabilities.importers.nvd import to_advisories
 
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 TEST_DATA = os.path.join(BASE_DIR, "test_data/nvd/nvd_test.json")
 
 
-class TestNVDImporter(TestCase):
```

Comment:
@TG1999 Just wondering, why have the tests been removed? Do we not need them anymore?

Reply:
@JonoYang these are old tests that are nor compatible with current model changes, I have removed the old ones and added new tests